cli/src/main/java/org/infinispan/cli/commands/rest/Roles.java

```diff
@@ -132,6 +132,33 @@ protected CompletionStage<RestResponse> exec(ContextAwareCommandInvocation invoc
       }
    }
 
+   @CommandDefinition(name = "update", description = "Updates an existing role")
```

Comment:
You need to add `Roles.Update.class` to the `@GroupCommandDefinition` at the top